### PR TITLE
Displays information in table format

### PR DESCRIPTION
**Pull request by DN376**
Displays information in table format

### CODE REVIEW

File: GetAPIData.py [hunk 1]
```diff
@@ -18,14 +19,22 @@
...
     try:
         response = session.get(url, params=parameters)
         apiData = json.loads(response.text)
-        return render_template('coin.html', data=apiData['data'])
+        coinHeader =  Coin("Coin", "Name","Price", "Market Cap")
+        for c in apiData['data']:
+          coinData = Coin(c['name'],
+          c['symbol'],
+          c['quote']['CAD']['price'],
+          c['quote']['CAD']['market_cap'])
```
Comment (trinhan-nguyen) [inline GetAPIData.py]: You may want to reformat it in a more standard way

File: coin.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Coin:
+    def __init__(self, name, symbol, price, marketCap):
+        self.name = name
+        self.symbol = symbol
+        self.price = price
+        self.marketCap = marketCap
```
Comment (trinhan-nguyen) [inline coin.py]: Good use of class

**Review by trinhan-nguyen — CHANGES_REQUESTED** [review]
Just a small thing, but should be changed

File: GetAPIData.py (hunk 1 above)
Comment (DN376) [reply GetAPIData.py]: updated

**Review by trinhan-nguyen — APPROVED** [review]

**Pull request merged**